bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/channel_types.xml

```diff
@@ -54,6 +54,20 @@
 				<option value="11">PV Wake-up Battery</option>
 				<option value="12">GEN Check</option>
 				<option value="13">GEN Run</option>
+				<option value="100">Waiting</option>
```

Comment:
This channel now combines both local and cloud values into one channel-type / list . 

I think there should be two variants of this channel-type, one for cloud, one for local, each holder their own set of the values.

Reply:
It's not really an issue because essentially they're the same but I will make it a separate thing type.